packages/graphql/src/schema-model/annotation/SubscriptionsAuthorizationAnnotation.ts

```diff
@@ -22,11 +22,11 @@ import type { GraphQLWhereArg } from "../../types";
 export const SubscriptionsAuthorizationAnnotationArguments = ["filter"] as const;
 
 export const SubscriptionsAuthorizationFilterEventRule = [
-    "CREATE",
-    "UPDATE",
-    "DELETE",
-    "CREATE_RELATIONSHIP",
-    "DELETE_RELATIONSHIP",
+    "CREATED",
+    "UPDATED",
+    "DELETED",
+    "RELATIONSHIP_CREATED",
+    "RELATIONSHIP_DELETED",
```

Comment:
this list of hardcoded strings is being duplicated more than 3 times across the code. Do you think it's worth modifying so that we have one source of truth for it?